### DISCUSSION TITLE
In the new bulk form, if I don't have the campaign ID, I can't add anything.

### DISCUSSION THREAD

**Discussion by shawchen123** [amzn/ads-advanced-tools-docs#194] [General]
In the old excel batch form, I can add new ad groups, new ASINs, new keywords, etc. at will. I only need to write the same name as the old ad campaign, but in the new ad batch table, I can't do that anymore, if I want to add something new, then I have to download the old bulk table file first to see what the old campaign ID and ad group ID are, I have to get that information before I can add the new Something, too bad, it's too much trouble. I want to know how I can fix this problem, I don't want to be too much trouble.

**Comment by DrRockett** [comment]
Hi! Sorry for the delayed reply. You are correct about needing the IDs to make updates to campaigns, ad groups, etc. It sounds like you are newer to the latest version, so I recommend [starting with this article](https://advertising.amazon.com/API/docs/en-us/bulksheets/2-0/overview-about-bulksheets) if you want to learn more about using it. [These videos about creating campaigns](https://advertising.amazon.com/API/docs/en-us/bulksheets/2-0/bulksheets-videos#create-campaigns) might also be helpful, and are pretty short. 

A bit more info: When you create new campaigns and ad groups, you will assign temporary IDs [to those entities and the "child" entities](https://advertising.amazon.com/API/docs/en-us/bulksheets/2-0/sp-entities) underneath. When you create the campaign, those IDs will become system-generated unique IDs, which you can get by downloading a bulk file. You will use those unique IDs to make updates to the campaigns later. 

I hope this information is helpful. I think if you start using the new version more, you'll see the benefits--you can make sweeping updates to campaign and ad group names, you can see robust performance metrics, and we are releasing new features to maintain parity with API functionality. Please reach back out if you have more questions! 

Thanks,
Danika

**Reply by shawchen123** [reply]
I certainly know how to use the new version of the bulk table file, I just want to say that the new version of the bulk table file is not as convenient as the old version, because when I want to update the advertisement, I need to download the already generated batch table file to get the advertisement ID, this means that I need to find the corresponding ID one by one, which is too troublesome, even if I will use simple excel functions to complete these tasks, but I want to say that the old version does not need to be so troublesome, I hope you can realize this a little.
